packages/fetchai/contracts/erc1155/contract.py

```diff
@@ -79,6 +90,13 @@ def create_token_ids(self, token_type: int, nb_tokens: int) -> Dict[int, int]:
 
         return self.token_ids
 
+    def generate_single_item_id(self, token_type: int, game_currency: int) -> int:
```

Comment:
There is no need for this function. You already have `create_token_id` with which you can generate a single token.